Author: Niklas Sorensson <nik@chalmers.se> (extracted from upstream git)
Approved: Ralf Treinen <treinen@debian.org>
Description: use exitcode to indicate error

Index: minisat+/Main.C
===================================================================
--- minisat+.orig/Main.C	2011-05-24 20:48:23.000000000 +0200
+++ minisat+/Main.C	2011-05-24 20:50:13.000000000 +0200
@@ -261,7 +261,7 @@
     reportf("*** TERMINATED ***\n");
     outputResult(*pb_solver, false);
     SatELite::deleteTmpFiles();
-    _exit(pb_solver->best_goalvalue == Int_MAX ? 0 : 10); }
+    _exit(0); }
 
 
 void printStats(BasicSolverStats& stats, double cpu_time)
@@ -324,7 +324,7 @@
         reportf("_______________________________________________________________________________\n");
     }
 
-    exit(pb_solver->best_goalvalue == Int_MAX ? 20 : (pb_solver->goal == NULL || opt_command == cmd_FirstSolution) ? 10 : 30);    // (faster than "return", which will invoke the destructor for 'PbSolver')
+    exit(0);    // (faster than "return", which will invoke the destructor for 'PbSolver')
 }
 
 
Index: minisat+/PbParser.C
===================================================================
--- minisat+.orig/PbParser.C	2011-05-24 20:48:23.000000000 +0200
+++ minisat+/PbParser.C	2011-05-24 20:50:13.000000000 +0200
@@ -282,7 +282,7 @@
             xfree(msg);
             if (opt_satlive && !opt_try)
                 printf("s UNKNOWN\n");
-            exit(opt_try ? 5 : 0);
+            exit(5);
         }else
             throw msg;
     }
